_includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md

```diff
@@ -14,10 +14,12 @@ To start feature integration, overview and install the necessary features:
 
 Activate the following plugins:
 
-| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
-| --- | --- | --- | --- |
-| StoreHttpHeaderApplicationPlugin | Gets store name from the Request header. | None | Spryker\Glue\StoresRestApi\Plugin\Application |
-| LocaleApplicationPlugin | Gets locale name from the Request header. | None | Spryker\Glue\ProductOptionsRestApi\Plugin\GlueApplication |
+| PLUGIN | SPECIFICATION                                                                                                                                  | PREREQUISITES | NAMESPACE                                                 |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------| --- |-----------------------------------------------------------|
+| StoreHttpHeaderApplicationPlugin | Gets store name from the Request parameter or Request header used for Glue Application. If both defined Request parameter has more priority. - | None | Spryker\Glue\StoresRestApi\Plugin\Application             |
```

Comment:
Please add a warning that `StoresRestApi` is deprecated, and we need to use `StoresApi` instead.